app/TracingStrategyAssets.js

```diff
@@ -0,0 +1,105 @@
+import { useTranslation } from 'react-i18next';
+import { Images } from './assets';
+import { config } from './COVIDSafePathsConfig';
+
+export function AppSpecificAssets() {
+  const { isGPS } = config;
```

Comment:
Does this work?

`import {isGPS} from '../blah';`

If so that's more conventional.

Change here and throughout